projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/tiled-gallery-rectangular.php

```diff
@@ -225,8 +226,9 @@ public function calculate_group_sizes( $row ) {
 		$group_widths_array = array();
 		foreach ( $row->groups as $group ) {
 			$group->height = $row->height;
+			$image_count   = is_countable( $group->images ) ? count( $group->images ) : 0;
```

Comment:
I worry about the same thing here. If `$image_count` is `0`, then the `$group->raw_width` will end up being negative.